common/types/loadbalancer.go

```diff
@@ -150,8 +150,8 @@ func NewLoadBalancer() *LoadBalancer {
 
 // K8sServiceNamespace is an abstraction for the k8s service + namespace types.
 type K8sServiceNamespace struct {
```

Comment:
Please move this to `pkg/policy/api` if you want to make it public.

Reply:
Moving this will induce an import cycle between `pkg/policy/api` and `common.types` packages, so I added this type also to API, so that both `common.types.K8sServiceNamespace` and `pkg/policy/api.K8sServiceNamespace` have the same underlying type.